docs/search.json

```diff
@@ -3,11 +3,13 @@
 ---
 
 [
-    {% for page in site.pages %}
+{% assign sorted_pages = site.html_pages | sort: 'title' %}
+{% for page in sorted_pages %}
+    {% if page.title != nil %}
```

Comment:
Should work the same in this case. I went with `nil` so that we are explicit, but I can change it.